docs/content/documentation/systems/Executioner/TimeStepper/framework/IterationAdaptiveDT.md

```diff
@@ -1,8 +1,59 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # IterationAdaptiveDT
 !syntax description /Executioner/TimeStepper/IterationAdaptiveDT
 
+## Description
+
+The `IterationAdaptiveDT` type of `TimeStepper` provides a means to adapt the time step size
```

Comment:
This `TimeStepper` provides a means to adapt the time step size based on the difficulty of the solution.